src/stories/atoms/Shape/Shape.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Shape } from 'stories/atoms/Shape/Shape';
+
+export default {
+  title: 'Design System/Atoms/Shape',
+  component: Shape,
+} as ComponentMeta<typeof Shape>;
+
+const Template: ComponentStory<typeof Shape> = (args) => <Shape {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  backgroundColor: '#c4c4c3',
+  shapeHeight: '250',
```

Comment:
Do you have a good reason for passing a `number` as a `string` other than it makes the string interpolation easier in the `.style` file? 

More importantly, if you're going to pass a `string`,  why not pass `250px` (include the `px` suffix)? This would create a more decoupled implementation because now we can pass props like `shapeWidth: '50%'` or `shapeHeight: "2em"`. 

The best front end devs are going to really thing about every piece of UI as a component, meaning that it should be as reusable as possible. Fewer dependencies === higher reusability

Reply:
Great point. I believe I chose a `string` because it was ultimately going to be concatenated with a `template literal`. As for the use of `px` in the `template literal`, I believe in my mind I was attempting to separate what I thought of as CSS syntax and what I thought of as the value going inside the CSS syntax. However, I can see how much better it would be to include the unit inside the `props` themselves. This makes the most sense because it allows more freedom and control in regards to units.